### PR TITLE
Document goFSH

### PR DESCRIPTION
**Pull request by ngfreiter**
This adds documentation for GoFSH, as well as a quick tutorial for goFSH.

### CODE REVIEW

**Comment by ngfreiter** [comment]
Making this a draft for now, because I want to test it out before merging, but after `gofsh` is released on npm.

**Comment by cmoesel** [comment]
I think we need some big fat disclaimers that GoFSH is a work in progress and is _not_ functionally complete -- especially for anything other than profiles and extensions (e.g., Code Systems, Value Sets, instances, etc).  Authors should review all GoFSH output before using it in their projects.

**Comment by ngfreiter** [comment]
> I think we need some big fat disclaimers that GoFSH is a work in progress and is _not_ functionally complete -- especially for anything other than profiles and extensions (e.g., Code Systems, Value Sets, instances, etc). Authors should review all GoFSH output before using it in their projects.

Good point, I've added disclaimers (inspired by those placed on the GoFSH README), to this documentation.

**Comment by ngfreiter** [comment]
Ok, this is now ready for review, I've tested it out using the `gofsh` cli, and it worked for me. I think one of @cmoesel or @jafeltra should test it on a mac, just to make sure.

File: content/docs/goFSH/installation/_index.md
Comment (cmoesel) [inline content/docs/goFSH/installation/_index.md:37]: Should we say that releases prior to 1.0.0 are considered pre-releases and may introduce breaking changes in minor version increments?

File: content/docs/goFSH/running/_index.md
Comment (cmoesel) [inline content/docs/goFSH/running/_index.md:28]: GoFSH takes more than SDs.  At the moment, it also processes Code Systems.  Perhaps, "GoFSH takes FHIR StructureDefinitions and other FHIR conformance definitions (e.g., ValueSets, CodeSystems) as input."

File: content/docs/goFSH/running/_index.md
Comment (cmoesel) [inline content/docs/goFSH/running/_index.md:34]: This sentence reads funny:
> For example, the [mCODE Implementation Guide](http://hl7.org/fhir/us/mcode/) contains definitions which are derived from the [US Core Implementation Guide](http://hl7.org/fhir/us/core/), see [mcode-cancer-patient](http://hl7.org/fhir/us/mcode/StructureDefinition-mcode-cancer-patient.html), which is derived from [us-core-patient](http://hl7.org/fhir/us/core/STU3.1/StructureDefinition-us-core-patient.html).

Comment (ngfreiter) [reply content/docs/goFSH/running/_index.md:34]: Thank you I'll be here all week

File: content/docs/Tutorials/goFSH/_index.md
Comment (cmoesel) [inline content/docs/Tutorials/goFSH/_index.md:36]: Might want to include a note:

> NOTE: The current pre-release of GoFSH does not always produce the most efficient FSH representation possible. For example, the `Laterality` extension produced by GoFSH contains slicing directives that are not strictly necessary, as this slicing is inferred when the `valueCodeableConcept` path is used in a rule.  While the explicit slicing directives are not _incorrect_, future versions of GoFSH will recognize them as redundant and omit them.

I mention this because @markkramerus noted this inefficiency and others may as well (or worse, beginning FSH users may thing it is necessary).

File: content/docs/Tutorials/goFSH/_index.md
Comment (cmoesel) [inline content/docs/Tutorials/goFSH/_index.md:49]: Typo: `generated` vs. `generate`

**Review by cmoesel — APPROVED** [review]
Perfecto!

**Pull request merged**
